lib/modelManagers/usermodel.js

```diff
@@ -22,6 +22,22 @@ class User {
     return createdRecord;
   }
 
+  /**
+   *
+   * @param {field} field The field to be searched.
+   * @static
+   * @memberof User
+   * @returns {object} The parameters from the query
+   */
+  static async findEmail(field) {
```

Comment:
The name `findName` is kind of misleading, a better name would be `findUserByEmail`. The parameter `field` can be better written as `email`